### PR TITLE
Garmin Biometrics and HRV

### PR DESCRIPTION
**Pull request by leite08**
Ticket: https://github.com/metriport/metriport-internal/issues/118

### Dependencies

- Upstream: https://github.com/metriport/metriport/pull/18
- Downstream: TBD

### Description

- Add Garmin Biometrics and HRV
- Fixes bug/leftover that added non-existing property to Garmin > Body

### Release Plan

- anytime

### CODE REVIEW

File: api/app/src/mappings/garmin/user.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }));
+  });
+};
+
+export const garminUserMetricsToBody = (
+  gBody: GarminUserMetrics
+): Biometrics | undefined => {
+  const bio: Biometrics = {
+    metadata: {
+      date: gBody.calendarDate,
```
Comment (Goncharo) [inline api/app/src/mappings/garmin/user.ts:34]: Should we use the query date to guarantee this is `YYYY-MM-DD`, or is this not an issue?

**Review by Goncharo — APPROVED** [review]
One small comment, not sure if it needs to be addressed, otherwise 🚢 🚢 🚢

File: api/app/src/mappings/garmin/user.ts (hunk 1 above)
Comment (leite08) [reply api/app/src/mappings/garmin/user.ts:34]: This comes from Garmin, I managed to add some validation with this + dayjs' `CustomParseFormat` ([source](https://day.js.org/docs/en/parse/is-valid)):

`  date: z.string().refine((v) => dayjs(v, ISO_DATE, true).isValid()),`

Comment (leite08) [reply api/app/src/mappings/garmin/user.ts:34]: Ended up adding this on this upstream PR's [commit](https://github.com/metriport/metriport/pull/18/commits/8809ba9ee2bd819015166899b6a3729e95538657).

**Pull request merged**